### PR TITLE
direcly show login screen on saml expiration

### PR DESCRIPTION
**Pull request by theScrabi**
fixes https://github.com/owncloud/android/issues/2238#issuecomment-396660454

### CODE REVIEW

File: src/com/owncloud/android/ui/activity/FileDisplayActivity.java [hunk 1]
```diff
@@ -56,6 +56,8 @@
 import com.owncloud.android.AppRater;
 import com.owncloud.android.MainApp;
 import com.owncloud.android.R;
+import com.owncloud.android.authentication.AccountAuthenticatorActivity;
```
Comment (davigonz) [inline src/com/owncloud/android/ui/activity/FileDisplayActivity.java]: Please delete this dependency and the one below since you are not using them

**Comment by davigonz** [comment]
@theScrabi please review the requested changes and rebase with master

File: src/com/owncloud/android/ui/activity/FileDisplayActivity.java [hunk 2]
```diff
@@ -56,6 +56,7 @@
 import com.owncloud.android.AppRater;
 import com.owncloud.android.MainApp;
 import com.owncloud.android.R;
+import com.owncloud.android.authentication.AuthenticatorActivity;
```
Comment (davigonz) [inline src/com/owncloud/android/ui/activity/FileDisplayActivity.java]: This is not being use either

**Comment by jesmrec** [comment]
SAML expirations redirects right now to the login view instead of the list of servers. 

Approved

**Pull request merged**